### ISSUE TITLE
Missing button text main screens (not running a train)

### ISSUE THREAD

**Issue by edgreenberg** [leezer3/OpenBVE#875]
### Description

Missing button text main screens (not running a train)

* When game is opened, the five buttons on the left properly have visible text (Start new game, REview, Etc.)
* On the route/train selection screen, the bottom left and right buttons are blank
* Customize controls has ALL blank buttons (seven of them)
* Options has all blank buttons 
* Package management has three blank buttons.  

Also, the four links (Visit, Check update, report problem and about) are blank. Just a gray area on the screen. I'm attaching screenshots. 

![OpenBVE_001](https://user-images.githubusercontent.com/151407/226136914-886f9be9-6584-452b-bd06-df27f66051b5.png)
![OpenBVE_002](https://user-images.githubusercontent.com/151407/226136917-422aaae4-87b5-48af-a164-3c27d98a941f.png)
![OpenBVE_003](https://user-images.githubusercontent.com/151407/226136922-945e4a1a-3009-4afe-b3ce-569bcacacd1a.png)
![OpenBVE_004](https://user-images.githubusercontent.com/151407/226136925-f682b756-656a-4550-9c7d-12dc26343c98.png)

### Reproduction

Happens before a route or train is selected

## Route

n/a

## Train

n/a

## Logs

Logs may be accessed through the 'Report Problem' button on the bottom left of the screen.
Please provide a log, and a crash-log if the game actually crashes.
[openBVE Bug Report18_03_2023.zip](https://github.com/leezer3/OpenBVE/files/11009527/openBVE.Bug.Report18_03_2023.zip)

### Related information 

* ubuntu 22.10
* OpenBVE-1.9.0.1.deb
* 01:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Cedar [Radeon HD 5000/6000/7350/8350 Series]
* Method of control (Keyboard, joystick, RailDriver) N/A

**Comment by leezer3** [comment]
Now that's wierd :/
Unfortunately, the main form under Mono has never been the most reliable, and WinForms under Linux are sadly essentially abandoned.

I'm wondering about text colouring being picked up from the system badly- IIRC Ubuntu still has a default black theme, so perhaps its something to do with that.
That also ties in with the main buttons working OK; I think the color on those is set manually so that the color change thing works OK.

**Comment by edgreenberg** [comment]
I will try messing with the various screen colors. Meanwhile, could I
trouble you to send a few screenshots so that I know what these buttons do?
I no longer have my old system. The old system was Fedora 31.

Outside of that, things are working very well under Ubuntu Mate 22.10. I
want to take a minute to thank you for all you do on this program. It is
really enjoyable to run the New York subways on it.

**Comment by Kenny-Hui** [comment]
Try changing your font in the options to something like "Noto Sans" or "Ubuntu"

![image](https://user-images.githubusercontent.com/28094366/226156387-f623d060-ab75-4051-95ac-8bba4dfeb202.png)

It seems that I broke it on an earlier PR related to UI Improvements.

I've made some changes to how these labels are displayed so it is no longer auto sized to prevent overflowing on some languages.

But on some fonts like Noto Sans CJK, the font height is higher than it should be.
Some label or button does not have enough height to show the font, and it will refuse to show the text if there's not enough height to display in the first place.

Almost all other fonts displays fine except Noto Sans/Serif CJK, and I think I somehow overlooked it.
I'll increase the height and make a PR later, thanks.
